### DISCUSSION TITLE
I want to make my portfolio site.

### DISCUSSION THREAD

**Discussion by sanjirou95** [community/community#108938] [Other Feature Feedback, Questions, & Ideas]
I want to make my portfolio site.
Please help me.

**Comment by Cizr** [comment]
the first thing I would ask you is do you have any programming knowledge? like HTML/CSS? or any front-end one? 2nd thing i would recommend start by watching youtube vids there are plenty showing how to make a portfolio, 3rd step take a github project (portfolio) and customize it the way u want it try understand the code and rebuild the structure

**Reply by sanjirou95** [reply]
I am not so good at HTML/CSS.

**Reply by Cizr** [reply]
trust me its not that hard to learn,just check youtube vids and again if u truly wanna master it do step 3, customize the project the same way u want just open as many projects as u can and try remove,adding changing and so on this will help you or gives u an idea on how things work goodluck brother

**Reply by sanjirou95** [reply]
okay, I'll try it. Thanks

**Comment by mo-cherif** [comment]
https://frontendmasters.com highly recommended if you want to develop your front-end engineering skills

**Reply by sanjirou95** [reply]
okay, I do. Thank you for your kind attention.

**Comment by MoldyLunchBox** [comment]
it depends on what you want to showcase on your portfolio.  if you are a web dev or want to be one. i suggest you follow a crash course online to learn things. then after that you could follow a portfolio tutorial on youtube there are tons. or if you wanna be cool. find a portfolio design on figma and try to turn it into a website.

**Reply by sanjirou95** [reply]
Oh, you're expert.
I appreciate your help.
